packages/website/ts/@next/pages/ecosystem.tsx

```diff
@@ -50,56 +53,67 @@ const benefits: BenefitProps[] = [
 ];
 
 export const NextEcosystem = () => (
-  <SiteWrap theme="light">
-    <Section isTextCentered={true}>
-        <Column>
-            <Heading size="medium" isCentered={true}>
-                Jumpstart your Business on 0x
-            </Heading>
-            <Paragraph size="medium" isCentered={true} isMuted={true} marginBottom="0">
-                The Ecosystem Acceleration Program gives teams access to a variety of services including funding, personalized technical support, and recruiting assistance. We created the Ecosystem Acceleration Program to bolster the expansion of both infrastructure projects and relayers building on 0x.
-            </Paragraph>
-            <LinkWrap>
-                <Button
-                    to="#"
-                    isWithArrow={true}
-                    isAccentColor={true}
-                >
-                    Apply now
-                </Button>
-                <Button
-                    to="#"
-                    isWithArrow={true}
-                    isAccentColor={true}
-                >
-                    Learn More
-                </Button>
-            </LinkWrap>
-        </Column>
-    </Section>
+    <SiteWrap theme="light">
+        <Section isTextCentered={true}>
+            <Column>
+                <Heading size="medium" isCentered={true}>
+                    Jumpstart your Business on 0x
+                </Heading>
+                <Paragraph size="medium" isCentered={true} isMuted={true} marginBottom="0">
+                    The Ecosystem Acceleration Program gives teams access to a variety of services including funding,
+                    dedicated technical support, and recruiting assistance. We created the Ecosystem Acceleration
+                    Program to bolster the expansion of both infrastructure projects and relayers building on 0x.
+                </Paragraph>
+                <LinkWrap>
+                    <Button
+                        href={constants.URL_ECOSYSTEM_APPLY}
+                        isWithArrow={true}
+                        isAccentColor={true}
+                        useAnchorTag={true}
```

Comment:
why not `target="_blank"`?

Reply:
The other link was called out as explicitly new window in the doc so wasn't sure about this one